pyseir/models/suppression_policies.py

```diff
@@ -120,11 +120,11 @@ def get_interventions(start_date=datetime.now().date()):
             if actual_date <= today:
                 rho.append(1)
             elif (actual_date - today).days <= 30:
-                rho.append(1.1 / R0)
+                rho.append(0.97 / R0)
             elif (actual_date - today).days <= 60:
-                rho.append(1.0 / R0)
+                rho.append(0.97 / R0)
             elif (actual_date - today).days <= 90:
-                rho.append(0.8 / R0)
+                rho.append(0.97 / R0)
```

Comment:
Seems like this should be pulled out into a variable?

Reply:
We could pull it out into config, but it is soon to be deprecated and they are just set by hand more or less.. So I dunno that it is worth it.